install.sh

```diff
@@ -77,16 +73,17 @@ __entry() {
 			--disable-mongo)
 				_INSTALL_MONGO=false
 				;;
-			-v|--version)
```

Comment:
We no longer want to support installing a different version of rita with a given installer. Each installer is tied one-to-one directly to a version of rita.